### PR TITLE
Remove vsts hack

### PR DESCRIPTION
**Pull request by forki**

### CODE REVIEW

File: src/Paket.Core/Dependencies/NuGetV2.fs [hunk 1]
```diff
@@ -331,73 +363,13 @@ let getDetailsFromNuGetViaODataFast auth nugetURL (packageName:PackageName) (ver
...
-            match raw with
-            | Some raw ->
-                let doc = getXmlDoc url raw
-                return parseODataEntryDetails(url,nugetURL,packageName,version,doc) |> ODataSearchResult.Match
-            | None -> return ODataSearchResult.EmptyResult }
-
-    async {
-        try
-            let! result =
-                // See https://github.com/fsprojects/Paket/issues/2213
```
Comment (matthid) [inline src/Paket.Core/Dependencies/NuGetV2.fs]: Can we keep that link?

**Comment by forki** [comment]
done

**Comment by matthid** [comment]
I don't think we need this anymore, my fallback cleanup PR includes this basically (I noticed this later)

**Comment by forki** [comment]
Cool. Thanks

Am 18.08.2017 22:11 schrieb "Matthias Dittrich" <notifications@github.com>:

**Pull request closed without merging**